faster_whisper/transcribe.py

```diff
@@ -574,7 +588,7 @@ def generate_with_fallback(
             # Recover the average log prob from the returned score.
             seq_len = len(tokens)
             cum_log_prob = result.scores[0] * (seq_len**options.length_penalty)
```

Comment:
Can you rename `cum_log_prob` to `cum_logprob` for consistency with `avg_logprob`?